core/store/src/main/java/mil/nga/giat/geowave/core/store/entities/GeowaveRowId.java

```diff
@@ -17,8 +17,7 @@
  * 
  */
 @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "private class only accessed internally")
-public class GeowaveRowId implements
-		NativeGeoWaveRow
+public class GeowaveRowId
```

Comment:
Yeah, GeoWaveRowImpl already manages the row ID. I did tackle many of the places where that (or AccumuloID, or whatever) were being used, but apparently not all of them. I need to go back and finish this up.